### ISSUE TITLE
Rename of organizational elements

### ISSUE THREAD

**Issue by yawlfoundation** [yawlfoundation/yawl#273]
```
Roles, capabilities, etc. need to be editable.
```

Original issue reported on code.google.com by `stephan....@googlemail.com` on 20 Feb 2009 at 9:08

**Comment by yawlfoundation** [comment]
```
What does this mean????
```

Original comment by `yawl.mic...@gmail.com` on 22 Feb 2009 at 3:17

**Comment by yawlfoundation** [comment]
```
So far it is not possible to rename organisational elements in the resource 
service.
They need to be removed and added again with a different name. 
```

Original comment by `stephan....@googlemail.com` on 22 Feb 2009 at 9:58

**Comment by yawlfoundation** [comment]
```
fixed for 2.1beta
```

Original comment by `yawl.mic...@gmail.com` on 2 Jun 2010 at 3:02
* Changed state: **Fixed**
